packages/base/src/Keys.js

```diff
@@ -245,4 +245,5 @@ export {
 	isPageDownAlt,
 	isPageUpShiftCtrl,
 	isPageDownShiftCtrl,
+	checkModifierKeys,
```

Comment:
This is a low level function and therefore not intended to be exposed to the control development layer. I'm not completely sure about the current use case, but you should for example create an `isEscapeShift` function or whatever fits best, and expose it instead of `checkModifierKeys`

Reply:
Done.